app/scripts/controllers/deployment.js

```diff
@@ -68,7 +68,7 @@ angular.module('openshiftConsole')
         var updateHPAWarnings = function() {
             HPAService.getHPAWarnings($scope.deployment, $scope.autoscalers, limitRanges, project)
```

Comment:
I added this filter to each of the deployment pages, but now think I probably should have just added a `HPAService.getHPAWarningsForResource()` method & eliminated the duplication.